petab/core.py

```diff
@@ -251,6 +251,18 @@ def to_float_if_float(x: Any) -> Any:
         return x
 
 
+def is_empty(val) -> bool:
```

Comment:
Isn't that pretty much https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.isnull.html?

Reply:
just checked: ``pd.isnull('') is False``, for None and nan we have True however. Maybe have '' or pd.isnull in the function?

Reply:
Okay, then either way is fine

Reply:
added pd.isnull now. could check for more eventualities, so better